_potd_131_enemies/ahriman.md

```diff
@@ -20,10 +20,11 @@ vulnerabilities:
 abilities:
   - name: Level 5 Petrify
     potency: n/a
-    description: 'untelegraphed conal AoE inflicting petrify (15s) - get behind
-    or get away'
+    description: 'untelegraphed conal AoE on random player inflicting petrify
+    (15s) - get behind or get away'
 notes:
   - 'Can be slowed if transfigured via Pomander of Witching'
+  - 'Stay close when in parties to avoid being caught out by random targeting'
```

Comment:
I think the cast time makes it easy to avoid either way, but aren't you more safe if far? I think it's a fairly short-range conal; otherwise "get away" (above) might be bad advice for avoiding it

Reply:
I had a case in a party recently where it chased a short distance after someone and put them smack in the center of the cone, catching them by surprise, so I'm speaking from that experience - if close, at least you're guaranteed a shorter path to safety, sort of like the Morbol sleep -> bad breath combo. But I can always remove this if you think it's questionable.

Reply:
(To clarify, I think "behind or away" is fine in the general case, and it's just the random targeting aspect that makes me want to say "prefer to stay close" when in parties.)

Reply:
Ah, makes sense. If I'm far away, I'm kiting, so already moving and going to easily avoid it, but I guess that wouldn't be the case in a party. Thanks for the explanation